### DISCUSSION TITLE
Frustratingly Precise Carousel Scroll Inputs

### DISCUSSION THREAD

**Discussion by michaeljoelphillips** [community/community#4498] [Mobile] [answered]
When browsing the "Explore" tab, I have found that swiping right/left on the carousels to scroll through different categories of new or trending repositories requires a frustratingly precise angle that I have to attempt multiple times before getting it right.  When I fail to get it to scroll, my inputs are mistaken for vertical inputs instead of horizontal inputs.

It's an odd UX problem to encounter, as the carousel doesn't meet my expectations of how I should be able to "grab" the thing and fling it right/left with fairly imprecise gestures.  Instead I feel like the component is too "tight" and requires too much effort and focus to scroll than it should.

I am experiencing this on a Google Pixel 3, not sure if other devices have the same issue.

**Comment by brianlovin** [comment, marked as answer]
Thanks for the note @michaeljoelphillips – agree, there is a bit too much "axis sensitivity" (for lack of a precise term here). I'll note this for our team.
